### PR TITLE
allow permanent delete of log_analytics_workspace_resource

### PR DESCRIPTION
**Pull request by lde**
Allow permanent delete of log_analytics_workspace_resource without go to recycle bin.

### CODE REVIEW

**Comment by lde** [comment]
refers issue #7282

**Review by katbyte — CHANGES_REQUESTED** [review]
Hi @lde - thanks for this pr, however due to the way terraform works we can't just put it into the schema. We'll have to add it to the providers feature block like `purge_soft_delete_on_destroy` or `delete_os_disk_on_delete`.

**Comment by lde** [comment]
Pr updated for use feature block.

File: azurerm/internal/provider/features.go [hunk 1]
```diff
@@ -26,6 +26,19 @@ func schemaFeatures(supportLegacyTestSuite bool) *schema.Schema {
 				},
 			},
 		},
+		"log_analytics_workspace": {
+			Type:     schema.TypeList,
+			Optional: true,
+			MaxItems: 1,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"permanent_delete": {
```
Comment (katbyte) [inline azurerm/internal/provider/features.go:35]: may i suggest
```suggestion
					"permanently_delete_on_destroy": {
```
or
```suggestion
					"delete_without_recovery": {
```

**Review by katbyte — CHANGES_REQUESTED** [review]
thanks @lde, Aside from one comment about the property name this is looking good

**Comment by lde** [comment]
changes applied.

**Comment by rick-ha** [comment]
hello - what is the status on this? I read that Ide made his changes - is there another underlying issue?

**Review by katbyte — APPROVED** [review]
Thanks @lde - LGTM now 👍

**Pull request merged**